processor/resourcedetectionprocessor/README.md

```diff
@@ -17,6 +17,18 @@ details of which are currently pending confirmation in the OpenTelemetry specifi
     * host.name
     * os.type
 
+* System detector custom configuration example:
+    ```yaml
+    detectors: ["system"]
+    system:
+        # A priority list of sources to get the hostname(**default**: `["dns", "os"]`)
+        hostname_sources: ["dns", "os"]
```

Comment:
```suggestion
        hostname_sources: ["os"]
```

Reply:
I have added the suggested changes. Please, review the same.